### ISSUE TITLE
The config for 4-GPU training

### ISSUE THREAD

**Issue by ycszen** [facebookresearch/detectron2#695]
## ❓Could I change the config of PanopticFPN when I train it on 4 GPUs?

The original config of PanopticFPN trains 16 batch size on 8 GPUs. Each GPU has 2 images.
Now I train it on 4 GPUs with no change. In other words, each GPU has 4 images. 
In my opinion, because the PanopticFPN adopts the 'FrozenBN' on ResNet backbone and 'GN' on the segmentation branch, we don't need to change the learning rate.

However, I got worse results with the original code and original 1x setting.

Results are below:
![image](https://user-images.githubusercontent.com/6136494/72363372-b9e4ed80-372f-11ea-8c72-782872c1b779.png)

**Comment by ppwwyyxx** [comment]
Are you using a version between Dec 19 - Dec 26, as noted in https://detectron2.readthedocs.io/notes/changelog.html#known-bugs-in-historical-versions?

**Comment by ycszen** [comment]
Yes... 
Thanks for your notion. 
Does the newest codes fix this bug?

**Comment by ppwwyyxx** [comment]
The bug exists between Dec 19 - Dec 26.
